lib/rules/color-hex-case/__tests__/index.js

```diff
@@ -164,3 +164,323 @@ testRule(
 		],
 	}),
 );
+
+testRule({
+	ruleName,
+	config: ['lower'],
+	syntax: 'css-in-js',
+	fix: true,
+	accept: [
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	stroke: "url(#gradiantA)",
```

Comment:
```suggestion
	stroke: "url(#AAA)",
```

Let's use a string that looks like an uppercase hex to make sure it's ignored.